src/main/java/uk/gov/hmcts/reform/laubackend/cases/controllers/CaseSearchController.java

```diff
@@ -144,8 +144,12 @@
             verifyRequestSearchParamsAreNotEmpty(inputParamsHolder);
             verifyRequestSearchParamsConditions(inputParamsHolder);
 
+            log.info("Case search REPOSITORY calling getCaseSearch...");
```

Comment:
Actually, this log message isn’t quite accurate we are calling the service not the repository.